engine/src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -887,4 +891,13 @@ class SlickDataAccess(databaseConfig: Config) extends DataAccess {
 
     runTransaction(action)
   }
+
+  private def materializeWorkflowDescriptor(id: WorkflowId, workflowSourceFiles: WorkflowSourceFiles): Future[WorkflowDescriptor] = {
+    import akka.pattern.ask
+    val materializedWfDescriptorFuture = (CromwellServer.validateActor ? MaterializeWorkflowDescriptorActor.MaterializeWorkflow(id, workflowSourceFiles)).mapTo[ValidationResult]
```

Comment:
I can use a similar thing here..ask with no timeouts.. But I'm still unsure of how to deal with a `Future[WorkflowDescriptor]` instead of a `WorkflowDescriptor` currently being needed by stuff in this file :confused: